docs/quick_tutorial/static_assets.rst

```diff
@@ -43,13 +43,18 @@ Steps
    .. literalinclude:: static_assets/tutorial/static/app.css
     :language: css
 
-#. Make sure we haven't broken any existing code by running the tests:
+#. Add a functional test that requires the newly added static file to be delivered:
+
+   .. literalinclude:: static_assets/tutorial/tests.py
+    :linenos:
```

Comment:
When it is appropriate and an obvious improvement, yes indeed, but not just for the sake of changing syntax. Sometimes we do want to include the entire file (e.g., "Create a new file and edit it as follows"). Other times we want to use `:emphasize-lines:` with `:lines:` and `:lineno-match:` such as when it's too much to include the entire `:pyobject:`. Holler if you have any doubts or questions. And thank you!